entries/s/subscribestar.com.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Patreon": {
```

Comment:
Please change the name to SubscribeStar.com

Reply:
I updated the name to SubscribeStar, since that seemed more in line with the other sites. If I should be including the root domain, let me know and I'll add it.

Reply:
I have corrected the name now, and I'll try to follow this rule for future PRs as well.